crates/cairo-lint-core/src/plugin.rs

```diff
@@ -82,6 +84,9 @@ impl AnalyzerPlugin for CairoLint {
                         &mut diags,
                     ),
                     SyntaxKind::StatementBreak => breaks::check_break(db.upcast(), node, &mut diags),
+                    SyntaxKind::ExprBinary => {
+                        bool_comparison::check_bool_comparison(db.upcast(), node, &mut diags)
```

Comment:
convert the node to `cairo_lang_syntax::node::ast::ExprBinary`  you can do `ExprBinary::from_syntax_node(node)`